src/main/scala/language.scala

```diff
@@ -0,0 +1,51 @@
+package piconot
+import scalafx.application.JFXApp
+import picolib.semantics._
+/**
+ * @author justisallen
+ */
+trait language extends JFXApp{
```

Comment:
I think the reasoning was that if we extend `JFXApp`, then the user can just extend `language` and not worry about extending `JFXApp`; but this probably brings about coupling concerns.